content/sensu-go/5.2/installation/auth.md

```diff
@@ -0,0 +1,442 @@
+---
+title: "Authentication"
+description: ""
+weight: 4
+version: "5.2"
+product: "Sensu Go"
+menu:
+  sensu-go-5.2:
+    parent: installation
+---
+
+- [Managing authentication providers](#managing-authentication-providers) (enterprise-only)
+- [LDAP authentication](#ldap-authentication) (enterprise-only)
+  - [Configuration](#configuring-ldap-authentication)
+  - [Examples](#configuration-examples)
+  - [Specification](#ldap-specification)
+
+Sensu requires username and password authentication to access the [Sensu dashboard][1], [API][8], and command line tool ([sensuctl][2]).
+For Sensu's [default user credentials][3] and more information about configuring Sensu role based access control, see the [RBAC reference][4] and [guide to creating users][5].
+
+In addition to built-in RBAC, [Sensu Enterprise][6] supports authentication using a Lightweight Directory Access Protocol (LDAP) provider.
+This guide describes the configuration process for enabling LDAP authentication with Sensu.
+
+## Managing authentication providers
+
+You can view and delete authentication providers using sensuctl.
+To set up LDAP authentication for Sensu, see the section on [configuring LDAP authentication](#configuring-ldap-authentication).
+
+**ENTERPRISE ONLY**: Authentication providers in Sensu Go require a Sensu Enterprise license. To activate your Sensu Enterprise license, see the [getting started guide][6].
+
+To view active authentication providers:
+
+{{< highlight shell >}}
+sensuctl auth list
+{{< /highlight >}}
+
+To view configuration details for an LDAP authentication named `default`:
+
+{{< highlight shell >}}
+sensuctl auth info ldap default
+{{< /highlight >}}
+
+To delete an LDAP authentication provider named `default`:
+
+{{< highlight shell >}}
+sensuctl auth delete ldap default
+{{< /highlight >}}
+
+## LDAP authentication
+
+Sensu Enterprise offers support for using a standards-compliant Lightweight Directory Access Protocol tool (like [OpenLDAP][7]) for authentication to the Sensu dashboard, API, and sensuctl.
+Sensu does not yet support Active Directory for LDAP authentication or other authentication providers.
+
+**ENTERPRISE ONLY**: LDAP authentication in Sensu Go requires a Sensu Enterprise license. To activate your Sensu Enterprise license, see the [getting started guide][6].
+
+### Configuring LDAP authentication
+
+**1. Write an LDAP configuration definition**
+
+Write an LDAP configuration definition using the [examples](#configuration-examples) and [specification](#ldap-specification) in this guide.
+
+**2. Apply the configuration using sensuctl**
+
+Log in to sensuctl as the [default admin user][3] and apply the configuration to Sensu.
+
+{{< highlight shell >}}
+sensuctl create --file filename.json
+{{< /highlight >}}
+
+You can verify that your LDAP configuration has been applied successfully using sensuctl.
+
+{{< highlight shell >}}
+sensuctl auth list
```

Comment:
Now `sensuctl authprovider list`

Reply:
We finally decided to use `sensuctl auth`.